common/lib/xmodule/xmodule/modulestore/tests/test_mixed_modulestore.py

```diff
@@ -436,6 +436,46 @@ def test_get_items(self, default_ms, max_find, max_send):
                 revision=ModuleStoreEnum.RevisionOption.draft_preferred
             )
 
+    @ddt.data((ModuleStoreEnum.Type.split, 2, False), (ModuleStoreEnum.Type.mongo, 3, True))
+    @ddt.unpack
+    def test_get_items_in_tree_only(self, default_ms, expected_items_in_tree, oprhan_in_items):
+        """
+        Test `in_tree_only` option helps in returning only those items which are present in course tree.
+        """
```

Comment:
Can you add to the doc string why `expected_items_in_tree` and `orphan_in_items` are parameterized?